Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsFragmentPresenter.java

```diff
@@ -534,8 +520,6 @@ private void addWiimoteSubSettings(ArrayList<SettingsItem> sl, int wiimoteNumber
 		}
 		catch (NullPointerException ex)
```

Comment:
I wonder if this try catch is needed at all now?

Reply:
I removed them, it looked like the only valid use of that was to cope with empty sections. A 'null' Setting is handled and causes the default to be used, which appears to be correct to me (or at least no behaviour changes from previously).

Any other null pointer exceptions that could occur (allocation failure? Incorrect object type passed as map key?) probably *should* crash and not be silently ignored